test/jest/components.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import {
+  VictoryArea,
+  VictoryAxis,
+  VictoryBar,
+  VictoryBoxPlot,
+  VictoryCandlestick,
+  VictoryChart,
+  VictoryContainer,
+  VictoryErrorBar,
+  VictoryGroup,
+  VictoryHistogram,
+  VictoryLabel,
+  VictoryLine,
+  VictoryPie,
+  VictoryPolarAxis,
+  VictoryScatter,
+  VictoryStack,
+  VictoryTooltip,
+  VictoryVoronoi
+} from "victory-native";
+import enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+enzyme.configure({ adapter: new Adapter() });
+
+const components = [
+  { component: VictoryArea, name: "VictoryArea" },
+  { component: VictoryAxis, name: "VictoryAxis" },
+  { component: VictoryPolarAxis, name: "VictoryPolarAxis" },
+  { component: VictoryBar, name: "VictoryBar" },
+  { component: VictoryBoxPlot, name: "VictoryBoxPlot" },
+  { component: VictoryCandlestick, name: "VictoryCandlestick" },
+  { component: VictoryChart, name: "VictoryChart" },
+  { component: VictoryContainer, name: "VictoryContainer" },
+  { component: VictoryErrorBar, name: "VictoryErrorBar" },
+  { component: VictoryGroup, name: "VictoryGroup" },
+  { component: VictoryHistogram, name: "VictoryHistogram" },
+  { component: VictoryLabel, name: "VictoryLabel" },
+  { component: VictoryLine, name: "VictoryLine" },
+  { component: VictoryPie, name: "VictoryPie" },
+  { component: VictoryScatter, name: "VictoryScatter" },
+  { component: VictoryStack, name: "VictoryStack" },
+  { component: VictoryTooltip, name: "VictoryTooltip" },
+  { component: VictoryVoronoi, name: "VictoryVoronoi" }
+];
+
+describe("Default render", () => {
+  components.forEach((C) => {
+    it(`should work for ${C.name}`, () => {
+      const wrapper = enzyme.shallow(React.createElement(C.component));
+      expect(wrapper).toHaveLength(1);
```

Comment:
For newer tests I'm also curious whether it's worth swapping enzyme for React testing library. Not an urgent concern right now, but it does seem to be the library more people are familiar with these days.

Reply:
I totally agree! I made a very brief attempt to get RN testing lib working, but ended up battling some things -- so I decided just to roll the old tests over, and try again with RNTL after this initial work is done.

The upshot: these enzyme tests, at the very least, will catch very egregious errors that would make components not mount at all.

Reply:
Sounds good! I agree that these tests still have value, and there's no need to do the extra work to get everything working right now.